omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.controller.js

```diff
@@ -98,11 +111,20 @@ export default class ObsAcrossEncountersController {
   }
 
   getConceptWithShortName(concept) {
-    angular.forEach(concept.names, (name) => {
-      if (name.conceptNameType == 'SHORT') {
-        concept.display = name.display;
+    let lastShortName = null;
+    for(let i=0; i < concept.names.length; i++) {
+      let conceptName = concept.names[i];
+      if (conceptName.conceptNameType == 'SHORT') {
```

Comment:
I'm not familiar with the code base, but why not use '===' for a stricter comparison?